client-registry-jempi/package-metadata.json

```diff
@@ -67,12 +70,12 @@
     "JEMPI_POSTGRES_MEMORY_LIMIT": "3G",
     "JEMPI_POSTGRES_MEMORY_RESERVE": "500M",
     "JEMPI_SESSION_SECURE": false,
+    "JEMPI_SESSION_DOMAIN_NAME": "localhost",
     "DOMAIN_NAME": "",
     "POSTGRESQL_PASSWORD": "postgres",
```

Comment:
POSTGRESQL_USERNAME and POSTGRESQL_DATABASE are missing here right ?

Reply:
The variables are in there

Reply:
https://github.com/jembi/platform/pull/261/files/5cbc541df9c0f3cce1a753663341bd745d99763e#diff-fc2d1132260ae5045f50e2d801eea9b429149663100c842d595a5de7c020bd8dR63-R64